packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +541,90 @@ void main() {
           ));
           handle.dispose();
         });
+
+        testWidgets('Default scroll physics for macOS/iOS', (WidgetTester tester) async {
+          final ReorderableListView listView = ReorderableListView(
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+            scrollController: ScrollController(),
+          );
+
+          await tester.pumpWidget(listView);
+          expect(listView.scrollController.position.physics, isInstanceOf<BouncingScrollPhysics>());
+        }, variant: const TargetPlatformVariant(<TargetPlatform>{TargetPlatform.iOS, TargetPlatform.macOS}));
+
+        testWidgets('Default scroll physics for android/fuchsia/linux/windows', (WidgetTester tester) async {
+          final ReorderableListView listView = ReorderableListView(
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+            scrollController: ScrollController(),
+          );
+
+          await tester.pumpWidget(listView);
+          expect(listView.scrollController.position.physics, isInstanceOf<BouncingScrollPhysics>());
+        }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
```

Comment:
This seems to be a repeat of the previous test, but with the wrong test name. We can just delete this test.